ahnlich/client/src/conn/db.rs

```diff
@@ -0,0 +1,40 @@
+use crate::conn::Connection;
+use crate::error::AhnlichError;
+use ahnlich_types::db::{DBQuery, ServerDBQuery, ServerResponse, ServerResult};
+use tokio::net::TcpStream;
+
+/// Simple TCP Connection to a host and port
+#[derive(Debug)]
+pub struct Conn {
```

Comment:
Rename as `DBConn`. Making sure to update any doc where it is currently shown as Conn